src/components/Modal/Translate.vue

```diff
@@ -20,7 +20,7 @@
   -->
 
 <template>
-	<NcModal size="large" @close="$emit('close')">
+	<NcModal :show.sync="isShow" size="large" @close="$emit('close')">
```

Comment:
what do we need the computed ```isShow``` for? Can't the ```show``` prop be used directly?

Reply:
It is because `show` is a prop. So, it is one-way-down binding data. It should not be used with `v-bind.sync`

Reply:
My understanding is that [sync](https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier) works like this:
```vue
<text-document v-bind:title.sync="doc.title"></text-document>
```
is the same as:
```vue
<text-document
  :title="doc.title"
  @update:title="doc.title = $event"
></text-document>
```

So it's two way binding as you said @luka-nextcloud - but you are not using the up binding from what i can tell. `isShow` as a computed cannot be assigned a new value either. `this.isShow = $event` won't work.

So from my point of view you could either use a one way bind without `.sync` or you'd need a data attribute to hold the show state and be able to assign to it.